worker/methods.go

```diff
@@ -79,12 +79,14 @@ type CodeSource map[string][]byte // map[pathInZip]code
 
 type Code struct {
 	Name           string        `json:"name"`
-	Runtime        string        `json:"runtime"`
-	FileName       string        `json:"file_name"`
+	Runtime        *string       `json:"runtime"`
```

Comment:
I was trying to avoid having the values sent to the API. The API might reject having mismatched combinations of parameters.